lib/keystore.js

```diff
@@ -57,17 +60,22 @@ function nacl_decodeHex(msgHex) {
 }
 
 
-var KeyStore = function(mnemonic, pwDerivedKey, hdPathString) {
-
-  this.defaultHdPathString = "m/0'/0'/0'";
+var KeyStore = function(mnemonic, pwDerivedKey, hdPathString, salt) {
```

Comment:
Yes, exactly: console.warn() would still make it backward compatible but guide the user to upgrade!